Dockerfile

```diff
@@ -1,3 +1,16 @@
+FROM golang:alpine AS BUILDER
+RUN mkdir /build 
+ADD main.go /build/
+ADD alert_rules/alert-rules.yml.tmpl /build/
+WORKDIR /build
+RUN apk update
+RUN apk add git
+RUN go get go.etcd.io/etcd/clientv3
+RUN go get github.com/sirupsen/logrus
+RUN go get github.com/spf13/viper
+RUN go build -o main .
+RUN CGO_ENABLED=0 GOOS=linux go build -a -installsuffix cgo -ldflags '-extldflags "-static"' -o main .
```

Comment:
@KarineValenca, can this alertmanager `sidecar` you created be a go mod? I think it would be cleanup our dockerfile.